packages/experimental/TabList/src/Tab/Tab.styling.ts

```diff
@@ -27,10 +27,11 @@ export const useTabSlotProps = (props: TabProps, tokens: TabTokens, theme: Theme
         justifyContent: 'center',
         padding: 1,
         backgroundColor: tokens.backgroundColor,
+        ...(!context.vertical ? { height: '100%' } : {}),
         ...borderStyles.from(tokens, theme),
       },
     }),
-    [tokens, theme],
+    [tokens, theme, context.vertical],
```

Comment:
Couldn't we just do `vertical` since we already pull it out of context in line 17?

Reply:
This was an oversight - we indeed can